### PR TITLE
wireguard: Don't allow a .0 device ip

### PR DESCRIPTION
**Pull request by doanac**
This is not a valid value

### CODE REVIEW

**Review by mike-sul — APPROVED** [review]

File: subcommands/devices/config_wireguard.go [hunk 1]
```diff
@@ -117,7 +117,7 @@ func findVpnAddress(factory string) string {
 
 	ips := factoryIps(factory)
 	for ip := serverIp + 1; ip < serverIp+10000; ip++ {
-		if _, ok := ips[ip]; !ok {
+		if _, ok := ips[ip]; !ok && byte(ip) != 0 {
```
Comment (mike-sul) [inline subcommands/devices/config_wireguard.go:120]: Do we allow `.1` device IP?

Comment (MrCry0) [reply subcommands/devices/config_wireguard.go:120]: Which network mask do we use for VPN? Shouldn't we test the host part of the address, not just the last byte?

Comment (doanac) [reply subcommands/devices/config_wireguard.go:120]: wireguard isn't putting devices on a subnet. they each just have a unique IP and route to one other in a mesh.

I was a little surprised, but .1 worked fine. It was just a .0 that tripped things up

Comment (MrCry0) [reply subcommands/devices/config_wireguard.go:120]: Oh, it's a mesh. Thanks, it's interesting, need to read more about WG :)

**Comment by doanac** [comment]
@Tim-Anderson - would you like to test a little more or would you rather me merge now?

**Comment by doanac** [comment]
@Tim-Anderson - another thing we might point out to users - this is just how *fioctl* assigns IPs through an API. A customer that had more control over their devices' MAC address could probably do something completely deterministic. (i.e if the low 16 bits of the MAC are unique for their fleet - map that onto a 10.0.x.x/16 subnet).

**Pull request merged**